Examples/Framework/include/ActsExamples/EventData/Measurement.hpp

```diff
@@ -494,6 +490,22 @@ class VariableMeasurementProxy
   }
 };
 
+template <MeasurementConcept OtherDerived>
+MeasurementContainer::VariableProxy MeasurementContainer::copyMeasurement(
+    const OtherDerived& other) {
+  VariableProxy meas = makeMeasurement(other.size(), other.geometryId());
+  meas.copyFrom(other);
+  return meas;
+}
+
+template <MeasurementConcept OtherDerived, std::size_t Size>
+MeasurementContainer::FixedProxy<Size> MeasurementContainer::copyMeasurement(
+    const OtherDerived& other) {
+  FixedProxy<Size> meas = makeMeasurement<Size>(other.geometryId());
+  meas.copyFrom(other);
+  return meas;
+}
```

Comment:
Should this have an assert (or even an exception) that `other.size() == Size`?

Reply:
I believe this is done in `copyFrom` / `fill`